feat(crl): add crl use example (#1851)
emqtt/emq-docs-cn: en_US/security/crl-ocsp-stapling.md

# CRL/OCSP Stapling

EMQX supports SSL/TLS, you can use X.509 certificate to enable client authentication and transport encryption.

Before the X.509 certificate expires, if the private key is exposed or the certificate is wrong, you need to revoke it to make sure your device is not being used illegally.
In this case, you can use the CRL or OCSP Stapling feature to achieve a more secure setup.

## CRL

The Certificate Revocation List (CRL) is a list maintained by the CA that contains the serial numbers and revocation times of certificates that have been revoked. 

> You can configure the request endpoint of the CA on EMQX and refresh the CRLs regularly. Different from the browser's mechanism for using CRLs, the MQTT client does not need to manage CRL and only needs to verify via EMQX during the connection handshake.

Reviewer: ```suggestion
You can configure the request endpoint of the CA on EMQX and refresh the CRLs regularly. Unlike CRLs on the browser, the MQTT client need not manage CRLs and EMQX is in charge of the verification during the connection handshake.
```

Reviewer: @wivwiv 要确认下是不是这个意思
